setup.py

```diff
@@ -39,7 +39,9 @@
 
 logger = logging.getLogger(__name__)
 
-version = '2.1.0.dev0'
+# We do not have to add any prefixes there - in our CI we automatically add dev prefix
+# Using ``--tag-build`` feature
+version = '2.1.0'
```

Comment:
Slight risk of confusing some users here - they may think main branch is this version, rather than a moving target.

Not big risk, but what has to change to keep dev in this version?

Reply:
Yeah. I think I solved it differently -> since we want to handle both cases in CI (we will for example push the 2.1.0 release without dev0 suffix when preparing rcs), I am checking if the prefix is already added and if it is (and is the same as we want to add) we do not add it again. But when they differ, it will fail. For example if we keep 2.1.0dev0 and try to build an RC from that, the package build will fail and complain that the two suffixes differ. In CI I will always try to add dev0.

This will give us extra protection around the release time - then we will simply not forget to remove `dev0` and keep the final version in the commits that will be used for RC preparation.